test/gc/binding.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+try {
+  module.exports = require('./build/Release/binding').ongc;
+} catch (e) {
+  module.exports = require('./build/Debug/binding').ongc;
```

Comment:
Not a bad idea but I decided to use the same logic we use in test/addons for the sake of consistency.  PTAL.